### ISSUE TITLE
[FR] Function for creating the navbar links for BS4?

### ISSUE THREAD

**Issue by maelle** [rstudio/rmarkdown#2222]
Via https://github.com/r-lib/pkgdown/pull/1702

Is there any work planned on generalizing `rmarkdown::navbar_links_html()` to newer Bootstrap version?

**Comment by cderv** [comment]
This function is used to create navbar for rmarkdown Website mainly. This is already supposed to work with **bslib** but if it needs adjustment, we should definitely do them. 

We'll need to look at examples of what the differences are and what is potentially broken, if anything. 

Is there any obvious reprex of any issue ?

**Comment by maelle** [comment]
No, just that the HTML that's expected is different based on BS changes. :sweat_smile:

**Comment by cderv** [comment]
Meaning the navbar does not work ? Or something else ?

**Comment by maelle** [comment]
To be honest I haven't tried.

It does not help that the examples in Bootstrap docs are slightly different for versions 3 and 4. :grimacing: 

https://getbootstrap.com/docs/4.0/components/navbar/ vs https://getbootstrap.com/docs/3.3/components/#navbar

But yeah I imagine that if one does not respect the HTML structure, the JS and CSS do not work as expected.

**Comment by cderv** [comment]
Ok we'll have a look. It seems R Markdown website have been working fine, so it is possible that **bslib** include a specific patch for making this work (`bs3-compat` layer)
